WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -459,6 +463,25 @@ && getIntent().getExtras().getBoolean(ARG_CONTINUE_JETPACK_CONNECT, false)) {
             initSelectedSite();
         }
         mSharedLoginResolver.tryJetpackLogin();
+
+        if (mJetpackBrandingUtils.shouldShowJetpackPoweredBottomSheet()
```

Comment:
`mJetpackBrandingUtils.shouldShowJetpackPoweredBottomSheet()` has 3 different checks,
https://github.com/wordpress-mobile/WordPress-Android/blob/4cced645e5f360c30561333a0654d8659b70e732/WordPress/src/main/java/org/wordpress/android/util/JetpackBrandingUtils.kt#L23-L25
I think these are not appropriate for our case.
We talked about an independent remote flag for this project with @guarani, and we thought we might want to turn off this project anytime, it'd be better to have an independent remote flag for just disabling notifications.